pkg/polaris/graphql/errors.go

```diff
@@ -58,6 +59,19 @@ func (e GQLError) isError() bool {
 	return len(e.Errors) > 0
 }
 
+func (e GQLError) isTemporary() bool {
+	if len(e.Errors) > 0 {
```

Comment:
Do we ever get more than one error? i.e. would it make sense to:
```
for _, err := range e.Errors {
  switch err {
    case strings.HasPrefix(err.Message, "Error checking ... "),
         strings.HasPrefix(err.Message, "UNAVAILABLE: .."):
      return true
  }
}
return false
```

Reply:
Not sure. I've only seen these two errors by themselves when they are temporary. I was thinking better be safe than sorry. 😄